DatadogInternal/Sources/NetworkInstrumentation/W3C/W3CHTTPHeadersWriter.swift

```diff
@@ -80,11 +88,20 @@ public class W3CHTTPHeadersWriter: TracePropagationHeadersWriter {
         ]
         .joined(separator: Constants.separator)
 
-        let ddtracestate = [
-            "\(Constants.sampling):\(sampled ? 1 : 0)",
-            "\(Constants.origin):\(Constants.originRUM)",
-            "\(Constants.parentId):\(String(spanID, representation: .hexadecimal16Chars))"
-        ].joined(separator: Constants.tracestateSeparator)
+        // while merging, the tracestate values from the tracestate property take precedence
+        // over the ones from the trace context
```

Comment:
Practically it's origin only at this moment?

Reply:
Yes but we have to careful when designing the merging thing.